Sources/LiveViewNativeAVKit/VideoPlayerView.swift

```diff
@@ -0,0 +1,232 @@
+///     def render(%{platform_id: :swiftui} = assigns) do
+///       ~SWIFTUI"""
+///       <VStack id="webview-example">
+///         <VStack>
+///           <%= @playback_time %>
+///         </VStack>
+///         <VideoPlayer
+///           autoplay="true"
+///           is-muted="is_muted"
+///           playback-time="playback_time"
+///           playback-time-update-interval="playback_time_update_interval"
+///           on-play="handle_play"
+///           on-pause="handle_pause"
+///           time-control-status="time_control_status"
+///           url="http://192.168.1.143:4000/videos/sample2.mp4"
+///           volume="volume" />
+///       </VStack>
+///       """
+///     end
+///   end
+/// ```
+///
+/// ## Attributes
+/// * ``autoplay``
+/// * ``url``
+///
+/// ## Bindings
+/// * ``isMuted``
+/// * ``playbackTime``
+/// * ``playbackTimeUpdateInterval``
+/// * ``timeControlStatus``
+///
+/// ## Events
+/// - ``pauseEvent``
+/// - ``playEvent``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct VideoPlayerView<R: RootRegistry>: View {
+    /// An event that is fired when the video player is paused.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @Event("on-pause", type: "click") private var pause
+
+    /// An event that is fired when the video player is played.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @Event("on-play", type: "click") private var play
+
+    /// A boolean indicating whether the video player is muted.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "is-muted") private var isMuted: Bool
+
+    /// The current playback time of the video player in seconds.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "playback-time") private var playbackTime: Float64
+
+    /// The interval at which the playback time is updated.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "playback-time-update-interval") private var playbackTimeUpdateInterval: Float64
+
+    /// The current time control status of the video player (playing, paused, etc.).
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    @LiveBinding(attribute: "time-control-status") private var timeControlStatus: String
+
+    /// If true, the video will play when the view appears.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    var autoplay: Bool = false
+
+    /// The URL of the video to play.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    var url: URL
+
+    @LiveContext<R> private var context
+    @ObservedElement private var element: ElementNode
+    @StateObject private var observer: VideoPlayerObserver<R>
+
+    init(element: ElementNode) {
+        if let value = element.attributeValue(for: "autoplay") {
```

Comment:
Can this be updated to use use the `@Attribute` property wrapper? If not, use `attributeBoolean(for:)` instead of `attributeValue(for:)`